packages/safe-tools-client/app/components/schedule-collapse-panel/index.css

```diff
@@ -0,0 +1,136 @@
+  height: var(--collapse-panel-summary-height);
+  padding: 0 var(--boxel-sp-xl);
+  border-radius: var(--collapse-panel-border-radius);
+  font-size: var(--boxel-font-size);
+  transition: margin-bottom var(--boxel-transition);
+}
+
+/* Adds margin to fake animation */
+.collapse-panel[open] .collapse-panel__summary {
+  margin-bottom: var(--collapse-panel-summary-anim-bottom-margin);
+}
+
+.collapse-panel__summary-collapsed {
+  display: inherit;
+  flex: inherit;
+  justify-content: space-between;
+  align-items: center;
+}
+
+/* Hide collapsed banner info */
+.collapse-panel[open] .collapse-panel__summary-collapsed {
+  display: none;
+}
+
+/* Use same style for "btns" and replace content once its open */
+.collapse-panel__summary-panel-button,
+.collapse-panel[open] .collapse-panel__summary::after {
+  font-weight: bold;
+  font-size: var(--boxel-font-size-sm);
+  content: 'Hide this';
+  cursor: pointer;
+  z-index: 1;
+}
+
+/* Remove bottom border and align "Hide this" to end */
+.collapse-panel[open] .collapse-panel__summary {
+  border-bottom-left-radius: unset;
+  border-bottom-right-radius: unset;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.collapse-panel__details {
+  display: flex;
+  flex-direction: row;
+  padding-left: var(--boxel-sp-xl);
+}
+
+.collapse-panel__details h2 {
+  margin-top: 0;
+  text-align: left;
+  font-size: 2.9rem;
+  font-weight: 300;
+}
+
+.collapse-panel__details p {
+  margin-top: 2px;
+  font-size: var(--boxel-font-size-sm);
+}
+
+.collapse-panel__details-content {
+  flex: 2;
+  padding-right: 150px;
+  padding-bottom: var(--boxel-sp-xl);
+}
+
+.collapse-panel__details-content ul {
+  display: flex;
+  gap: 80px;
+  justify-content: space-between;
+  list-style: none;
+  padding: 0;
+  width: 80%;
+}
+
+.collapse-panel__details-content ul li h4 {
+  margin: 0;
+  padding-top: var(--boxel-sp-xxxs);
+}
+
+.collapse-panel__details-overlay {
+  flex: 1;
+  /* Adds negative margin based on summary height minus the animation margin */
+  margin-top: calc((var(--collapse-panel-summary-height) + var(--collapse-panel-summary-anim-bottom-margin)) * -1);
+  border-radius: 0 var(--collapse-panel-border-radius) var(--collapse-panel-border-radius) 0;
+  background-color: var(--boxel-light-100);
+  opacity: 0.95;
+  padding: var(--boxel-sp-xxxl) var(--boxel-sp-xxl);
+  padding-bottom: var(--boxel-sp-lg);
+}
+
+.collapse-panel__details-overlay h3 {
+  font-size: var(--boxel-font-size-lg);
+  text-align: left;
+}
+
+/* Remove unstyled numbers */
+.collapse-panel__details-overlay ol {
+  margin-left: -2.5rem;
+  list-style: none;
+}
+
+/* Keep track of number */
+.collapse-panel__details-overlay ol li {
+  counter-increment: item;
+  padding-bottom: var(--boxel-sp);
+}
+
+/* Style ol numbers */
+.collapse-panel__details-overlay ol li:before {
+  content: counter(item);
+  color: var(--boxel-teal);
+  padding-right: var(--boxel-sp);
+  font-weight: 800;
+}
+
+.collapse-panel__details-overlay ol li h4 {
```

Comment:
I think using a class to target a specific element is preferable when we have control of the template, it makes it clear which things we're targeting.

type selectors can match things that we don't intend to match, especially as a component evolves.

Reply:
I thought it was okay for this case, since it's inside the .collapse-panel__details-overlay, but I understand it might be safer to use a class